configure.ac

```diff
@@ -2358,8 +2359,16 @@ AC_ARG_WITH([libdpdk],
 )
 
 if test "x$with_libdpdk" != "xno"; then
+  PKG_CHECK_MODULES(DPDK, [libdpdk],
+    [LIBDPDK_LIBS="$DPDK_LIBS"],
+    [LIBDPDK_LIBS="-ldpdk"])
   if test "x$LIBDPDK_CPPFLAGS" = "x"; then
-    LIBDPDK_CPPFLAGS="-I/usr/include/dpdk"
+    if test "x$DPDK_CFLAGS" != "x"
+    then
+      LIBDPDK_CPPFLAGS="$DPDK_CFLAGS"
```

Comment:
DPDK_CFLAGS can contain compiler flags that don't belong in _CPPFLAGS. Can you introduce an extra variable?